### PR TITLE
KNOX-2209 - Improve logging for Knox token handling

### PR DESCRIPTION
**Pull request by pzampino**
## What changes were proposed in this pull request?

Added logging to the Token and TokenState services.

## How was this patch tested?

Manually tested by way of visual inspection of logs.

### CODE REVIEW

File: gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java [hunk 1]
```diff
@@ -384,4 +391,9 @@ private long getExpiry() {
     }
     return expiry;
   }
+
+  private String getTopologyName() {
+    return (String) context.getAttribute("org.apache.knox.gateway.gateway.cluster");
```
Comment (risdenk) [inline gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java]: nit: Would be good to make this a constant. I ran into this looking into something else. It was NOT clear that this attribute was the topology name.

File: gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceMessages.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 import org.apache.knox.gateway.i18n.messages.Messages;
 import org.apache.knox.gateway.i18n.messages.StackTrace;
 
-@Messages(logger="org.apache.knox.gateway.service.knoxsso")
+@Messages(logger="org.apache.knox.gateway.service.token")
```
Comment (risdenk) [inline gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceMessages.java]: Might make sense to match the package name? `knoxtoken`

File: gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java (hunk 1 above)
Comment (pzampino) [reply gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java]: This is why I put it in a method that makes it more clear (aside from the obvious benefit of re-use). I can create a constant for it. Are you suggesting a constant in this class, or somewhere else for broader consumption?

File: gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceMessages.java (hunk 2 above)
Comment (pzampino) [reply gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceMessages.java]: I can buy that. I was surprised to see that it had been knoxsso :-O

File: gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java (hunk 1 above)
Comment (risdenk) [reply gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java]: Broader consumption - doesn't have to be as part of this change. The string `"org.apache.knox.gateway.gateway.cluster"` is being used to insert the topology value into the context.

Comment (pzampino) [reply gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java]: +1

**Pull request merged**